### ISSUE TITLE
AOB Review - Data Cleanup for Contractor Pick List

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#3500]
### 8/13/20 AOB Meeting

**People record type ‘City Owner’ Info:**
-	ATD uses this people type as a solution/workaround for entities that are performing ROW work, but do not require a ROW license because they’re COA or a franchise. 
-	Currently there are 433 ‘City Department’ people records. Only a small fraction of those represent entities that do not require a ROW license.
-	This causes the ROW portal applications to display hundreds of inaccurate ‘valid contractors’ for applicants to select from
-	This may also lead to ROW issuing a permit without a valid contractor listed. 
-	Our goal is to only populate valid contractor records for the portal applicants
**Questions for AOB:**
-	Is anyone using ‘City Department’ for a specific business need?
-	If we cleaned up ‘City Department’ records to only those that ROW needs, would that have negative impacts?
-	If cleaning up the ‘City Department’ records is not an ideal solution, we’ll need to create a new people type. Any objections?

**Comment by TracyLinder** [comment]
### Meeting Notes:

- The only possible concern is with Code Folders, who have used 'City Department' in the past, but aren't anymore. 
- Code requested a data pull of all folders related to a people record type 'City Department' 
    - Hoping @licarir can help w/ this

Next Steps:
- [ ] Design the solution
    - Identify all records that should not be 'City Department'
    - Update their people record type to **'TBD'**
- [ ] Send out data pull to department spocs who's folders show up in the data. 
    - Ask them if they approve of updating the people type on those records to **'TBD'**

**Comment by TracyLinder** [comment]
### Email to Shawn

Hi Sean,

This is a follow up to an AOB item I spoke on regarding cleaning up the ‘City Department’ people records in AMANDA. Attached is a list of your folders that have ‘City Department’ records attached to the people tab. 

Our ask of you: Do you have any concerns with us cleaning up the ‘City Department’ people records in AMANDA? In practice this would mean updating the people type of about 400 records from ‘City Department’ to another type (TBD). This would not affect any records on any of your folders; updating the people type on the people table does not update the people type on the folder people table. 

Hoping this makes sense, but let me know if you’d prefer a call to discuss.

**Comment by TracyLinder** [comment]
@TracyLinder add the email from Sean & close this.

**Comment by TracyLinder** [comment]
**Email From Shawn approving the solution:**

Hi Tracy,

Makes sense. I don’t have any concerns. The most recent ones I looked at should’ve been using our “City Owned SPOC” People Type rather than “City Department”. I just manually updated the couple records that had been created in the past couple years.  I don’t see any negative impact by changing the People Type on all the other people records for CC/CV.  The CR folder type is Complaint/Requests for Animal Control, so we don’t own this one.

Thanks!

 
Shawn Lincoln, PMP
Acting IT Supervisor Senior
City of Austin, Code Department
P: 512-974-9029 | E: shawn.lincoln@austintexas.gov
“Preserving Austin’s Quality of Life.”
